pypechain/render/init_test.py

```diff
@@ -25,15 +25,17 @@ def test_render_init(self, snapshot):
 
         # TODO: add return types to function calls
 
-        file_names = [
-            "ContractFile1",
-            "ContractFile2",
-            "ContractFile3",
+        file_outputs = [
+            RenderOutput(filename="ContractFile1", definitions=["ContractFile1Contract"]),
+            RenderOutput(filename="ContractFile2", definitions=["ContractFile2Contract"]),
+            RenderOutput(filename="TypesFile", definitions=["Struct1"]),
+            RenderOutput(filename="TypesFile", definitions=["Struct2"]),
         ]
 
+        # To avoid having to regenerate snapshots per version bump, we hard code the test version to 0.0.0 here
```

Comment:
I thought this was in place so that CI would render new snapshots for testing. Wouldn't that break now?

Reply:
This is due to the newest update that includes the version number in generated files, which breaks snapshots if the version gets bumped. Now snapshots is independent of version number since we hard code it to a static number in this case

Reply:
To rephrase, we don't compare snapshots against actual generated files, but instead, compare only the function that's generating the init file